### DISCUSSION TITLE
Show Dirs button

### DISCUSSION THREAD

**Discussion by Sekpion** [lllyasviel/stable-diffusion-webui-forge#412] [Q&A]
Does anyone know how I can integrate the "show Dirs" button into this interface? just as it appears in the image.
![2024-02-25 08_03_54-Stable Diffusion - Brave](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/138418447/bb722c4a-7f1a-499c-9734-36a8e28517d4)

**Comment by catboxanon** [comment]
This was removed upstream in favor of a toggleable tree view. https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/14588

**Comment by Dasor92** [comment]
is there a way to roll back having folders? When you have more than 2000 loras the new tree view is unusable. Also, the "change preview" button doesn't update automatically.

**Comment by AlluraHikaru** [comment]
This is one of the worst upstream changes, very awkward implementation. I just type the lora folder path manually in search now, it was so much easier before.

**Comment by freecoderwaifu** [comment]
If you use Civitai Helper then disable it, that's what might be giving you a bad experience, since it causes noticeable lag and slowdown, and that's on the extension's side. Tree View is perfectly usable as is, even with tens of thousands of Loras, the only thing it really needs is being able to resize the left and right panels.

Looking more into it, it seems the recent updates to this fork have fixed the lag, so updating to this fork (which is the one listed in the Extensions downloader list) might help too.

https://github.com/zixaphir/Stable-Diffusion-Webui-Civitai-Helper

**Reply by Dasor92** [reply]
Actually it’s not “unusable” because of its speed but because there are no way to open change folder quickly when you have more then 20 Loras in a folder just because when it’s open the folder below it becomes unclickable unless you scroll or close the folder. It’s just bad design
